ui/components/ui/account-list/account-list.js

```diff
@@ -28,14 +38,22 @@ const AccountList = ({
     selectedAccountScrollRef.current?.scrollIntoView({ behavior: 'smooth' });
   }, [selectedAccounts]);
 
+  const shortenAddress = (address, chars = 5) => {
+    if (!address) {
+      return '';
+    }
+    return `${address.slice(0, chars + 2)}...${address.slice(-chars)}`;
```

Comment:
We already have a utility function for this

Reply:
Do you have a reference to the use case so I can reuse it?

Reply:
Refactored the UI in a different way that does not need this anymore.